examples/interactive_play.py

```diff
@@ -335,6 +339,7 @@ def play_env(env, args, config):
     all_obs = []
     total_reward = 0
     all_arm_actions = []
+    agent_to_control = "AGENT_0"
```

Comment:
I would better save agent id, instead of config node. Then it will be easy to switch to next agent.